src/server/adminDeleteUser.ts

```diff
@@ -2,30 +2,78 @@
 
 import { getSupabaseAdminAuthClient } from "@/supabaseAdminAuthClient";
 import { authenticateAsAdmin } from "@/server/authenticateAdminUser";
-import { logInfoReturnLogId } from "@/logger/logger";
+import { logErrorReturnLogId, logInfoReturnLogId } from "@/logger/logger";
+import { AuditLog, sendAuditLog } from "@/server/auditLog";
+import { getSupabaseServerComponentClient } from "@/supabaseServer";
 
-export type DeleteUserErrorType = {
-    error: Record<string, string> | null;
-};
+type DeleteUserErrorType =
+    | "Failed to authenticate as admin"
+    | "Failed to get user id from profiles"
+    | "Failed to delete user";
```

Comment:
These tend to be camelCase or all lowercase, since they serve as a way for the code to differentiate what kind of error it is returning, not a user friendly error message. (This function shouldn't be concerned about what error message is displayed, but about returning the error data back to the caller)

e.g. `"failedToAuthenticateAsAdmin"`